sources/api/pluto/src/main.rs

```diff
@@ -157,9 +192,62 @@ fn get_max_pods(client: &Client, session_token: &str) -> Result<String> {
     error::NoInstanceTypeMaxPods { instance_type }.fail()
 }
 
-fn get_cluster_dns_ip(client: &Client, session_token: &str) -> Result<String> {
+/// Returns the cluster's DNS IPV4 address. First it attempts to call EKS describe-cluster to find
+/// the `serviceIPv4CIDR`. If that works, it returns the first `*.10` address. If the EKS call is
```

Comment:
I'm not sure I understand the "first `*.10` address" part of this comment - is it "10.*"?  If so, that only seems to apply if we fail to get the data from EKS and we're down on line 261, right?